mathics/builtin/lists.py

```diff
@@ -1970,17 +1971,29 @@ def apply_pattern(self, items, sel, pattern, evaluation):
 class Cases(Builtin):
     """
     <dl>
-    <dt>'Cases[$list$, $pattern$]'
-        <dd>returns the elements of $list$ that match $pattern$.
-    <dt>'Cases[$list$, $pattern$, $ls$]'
-        <dd>returns the elements matching at levelspec $ls$.
+      <dt>'Cases[$list$, $pattern$]'
+      <dd>returns the elements of $list$ that match $pattern$.
+
+      <dt>'Cases[$list$, $pattern$, $ls$]'
+      <dd>returns the elements matching at levelspec $ls$.
+
+      <dt>'Cases[$list$, $pattern$, Head->$bool$]'
```

Comment:
typo: `Heads`

Reply:
Thanks @axkr !  Should be fixed in https://github.com/mathics/Mathics/commit/834c7812c1feed871092ca8a6ded0a7da8c1f529